plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergGcsConnectorSmokeTest.java

```diff
@@ -190,6 +190,14 @@ public void testRenameSchema()
                 "Hive metastore does not support renaming schemas");
     }
 
+    @Test(timeOut = 150_000, invocationCount = 4)
+    @Override // Overridden to increase the timeout because GCS write speed is slower than other object storages
```

Comment:
how long does the test take on S3?
where is the GCS test bucket located? are we writing with a high latency?